dist/utils/baseToDisplay.d.ts

```diff
@@ -1,2 +1,2 @@
-import BN from 'bn.js';
-export declare const baseToDisplay: (base: string | BN, decimals: number) => string;
+import BN from 'bn.js';

```

Comment:
The diff in all files is `^M` for line endings.

From Stackoverflow:
`This is likely because the user who committed the file was using windows, which uses \r\n for its line endings. Linux uses only \n, hence you see the ^M at the end of the line.`